### PR TITLE
TRITON-2426 triton-origin-x86_64-23.4.0 image

### PR DESCRIPTION
**Pull request by jperkin**

### CODE REVIEW

**Review by danmcd — APPROVED** [review]
Optional, check with @bahamat :

- Add the TRITON- ticket to the summary that covers Joyent -> TritonDataCenter; I can't remember which ticket number it is, but you do fix this repo in that regard as well.

**Comment by jperkin** [comment]
> Optional, check with @bahamat :
> 
> * Add the TRITON- ticket to the summary that covers Joyent -> TritonDataCenter; I can't remember which ticket number it is, but you do fix this repo in that regard as well.

I couldn't find a ticket for this, and the repositories I checked just have their own separate changes. It's quite possible I missed it though, searching for "Joyent" etc in tickets is obviously needle-in-a-haystack territory ;-)

**Comment by danmcd** [comment]
> I couldn't find a ticket for this, and the repositories I checked just have their own separate changes. It's quite possible I missed it though, searching for "Joyent" etc in tickets is obviously needle-in-a-haystack territory ;-)

I did say optional.  :)  TRITON-2423 and/or TOOLS-2574 for manta URLs, TRITON-2304 for github repos.  They ARE optional, and maybe only need related-to links in the ticket.  My +1 stands!

File: README.md [hunk 1]
```diff
@@ -276,16 +292,15 @@ How to release a new triton-origin image:
    a few extra things are required:
 
    - Add entries to the "compatibility" tables in the "sdcnode compatibility
-     with triton-origin images" and "Joyent Jenkins agent labels compatibility
+     with triton-origin images" and "Jenkins agent labels compatibility
```
Comment (bahamat) [inline README.md:295]: This doesn't have an end quote after `"Jenkins`

**Review by bahamat — CHANGES_REQUESTED** [review]
Everything else looks good

File: README.md (hunk 1 above)
Comment (jperkin) [reply README.md:295]: It's there, it's just on the next line which was unmodified.

Comment (bahamat) [reply README.md:295]: Oh, I see it. Sorry. I swear I had looked for it before.

**Review by bahamat — APPROVED** [review]

**Pull request merged**